### ISSUE TITLE
Review access rigths on collective organization?

### ISSUE THREAD

**Issue by gforcada** [collective/Collective#1744]
Hi all,

I noticed that quite a few repositories in this `collective` organization are __forks__ from other organizations but, unlike `eea.facetednavigation` are totally random/unrelated to plone.

Looking at the [organization audit logs](https://github.com/organizations/collective/settings/audit-log) it mostly seems to be a UX problem on GitHub UI I would say 🍀 

Should we revisit who can create forks on this organization? Maybe, just like one has to ask to get invited in this organization, the same could be done for forks? 🤔 

There are left __+200__ forks in this organization, after deleting around +20 already.

**Comment by jensens** [comment]
Cleaning this up is a good idea, also cleaning up the access. How does restricting access works for forks?

**Comment by jensens** [comment]
Is there a way to list forks only?

**Comment by gforcada** [comment]
Yes: https://github.com/orgs/collective/repositories?q=&type=fork&language=&sort=

As to restrict I have to have a look, but I will do in the following days if nobody beats me at that 😃

**Comment by gforcada** [comment]
Either my search skills are not that good, or once you are a member of an organization you can actually fork anything into it 🤷🏾
